src/JWT.ts

```diff
@@ -322,31 +319,37 @@ export async function resolveAuthenticator(
   if (!types || types.length === 0) {
     throw new Error(`No supported signature types for algorithm ${alg}`)
   }
-  const doc: DIDDocument = await resolver.resolve(issuer)
-  if (!doc) throw new Error(`Unable to resolve DID document for ${issuer}`)
+  const result: DIDResolutionResult = await resolver.resolve(issuer, { accept: DID_JSON })
```

Comment:
This library technically supports the JSON and JSON-LD representations. If a resolver defaults to CBOR that would break this library. Unfortunately there doesn't seem to be a way to specify multiple ways strings here.